packages/contracts/test/voting/token-voting.ts

```diff
@@ -123,22 +127,103 @@ describe('TokenVoting', function () {
           supportThreshold,
           minParticipation,
           0,
+          minProposerVotingPower,
           governanceErc20Mock.address
         )
       ).to.be.revertedWith(customError('VoteDurationZero'));
     });
   });
 
   describe('Proposal creation', async () => {
-    supportThreshold = pct16(50);
-    minParticipation = pct16(20);
+    it('reverts if the user is not allowed to create a proposal', async () => {
+      minProposerVotingPower = 1;
+
+      await voting.initialize(
+        dao.address,
+        supportThreshold,
+        minParticipation,
+        minDuration,
+        minProposerVotingPower,
+        governanceErc20Mock.address
+      );
+
+      await governanceErc20Mock.mock.getPastTotalSupply.returns(1);
+      await governanceErc20Mock.mock.getPastVotes.returns(0);
+
+      await expect(
+        voting.createProposal(
+          dummyMetadata,
+          [],
+          startDate,
+          endDate,
+          false,
+          VoteOption.None
+        )
+      ).to.be.revertedWith(
+        customError('VoteCreationForbidden', signers[0].address)
+      );
+
+      await governanceErc20Mock.mock.getPastVotes.returns(1);
+      await expect(
+        voting.createProposal(
+          dummyMetadata,
+          [],
+          startDate,
+          endDate,
+          false,
+          VoteOption.None
+        )
+      ).to.not.be.reverted;
```

Comment:
doesn't the `reverted` bit need to be invoked `()` so that it does something?

Reply:
No, that's how it is supposed to work. See https://ethereum-waffle.readthedocs.io/en/latest/matchers.html#revert